UsersApp/View Controllers/UsersViewController.swift

```diff
@@ -36,30 +43,30 @@ class UsersViewController: UIViewController {
                 DispatchQueue.main.async {
                     self.usersTableView.reloadData()
                 }
-            } catch {
-                print("Error! Can't fetch the users.")
+            } catch let error as NetworkError {
+                print("Network error: \(error.localizedDescription)")
+                let alert = UIAlertController(title: "Could not fetch the users!", message: "\(error.localizedDescription)", preferredStyle: .alert)
+                alert.addAction(UIAlertAction(title: NSLocalizedString("OK", comment: "Default action"), style: .default, handler: { _ in
```

Comment:
instead of "NSLocalizedString("OK", comment: "Default action")", you can use just a simple string "Ok" ;)